packages/styles/scss/patterns/sections/card-section/_card-section.scss

```diff
@@ -6,7 +6,8 @@
 //
 @import '../../../components/link-with-icon/link-with-icon';
 @import '../../../globals/imports';
-@import '../../../globals/utils/aspect-ratio';
+// @import '../../../globals/utils/aspect-ratio';
+@import '../../../globals/utils/ratio-calc';
```

Comment:
If I'm not mistaken your new `ratio-calc` does the same thing the original `aspect-ratio` does.